code/modules/clothing/neck/_neck.dm

```diff
@@ -199,11 +199,11 @@
 	var/tagname = null
 
 /obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
-	if(ishuman(M))  // yogs start - lets catpeople wear collars
+	if(ishuman(M))
 		var/mob/living/carbon/human/T = M
-		if(iscatperson(T))
-			return TRUE
-		return FALSE // yogs end
+		if(!ishumanbasic(T))
+			return ..()
+		return FALSE
 	return ..()
```

Comment:
```	if(!ishumanbasic(T))
		return ..()
	return FALSE```
Replace this whole proc with this

Reply:
does that check not also occur for trying to equip collars to animals

Reply:
or I might be retarded